internal/adapter/handlebars/helpers/getdate.go

```diff
@@ -0,0 +1,23 @@
+package helpers
+
+import (
+	"time"
+
+	"github.com/aymerick/raymond"
+	"github.com/mickael-menu/zk/internal/util"
+	"github.com/pkg/errors"
+	naturaldate "github.com/tj/go-naturaldate"
+)
+
+// RegisterGetDate registers the {{getdate}} template helper to use the `naturaldate` package to generate time.Time based on language strings.
+// This can be used in combination with the `date` helper to generate dates in the user's language.
+// {{date (get-date "last week") "timestamp"}}
+func RegisterGetDate(logger util.Logger) {
+	raymond.RegisterHelper("get-date", func(natural string) time.Time {
+		date, err := naturaldate.Parse(natural, time.Now())
```

Comment:
Sorry I missed this in my first review, but you can use the helper [`TimeFromNatural()`](https://github.com/mickael-menu/zk/blob/c21c4fc21f7fe75f05d3fc3d09e74ffed41ba2ef/internal/util/date/date.go#L39) (which is part of this project) instead of using directly `naturaldate.Parse()`. This way it will be able to parse more formats.